backend/api/Services/MissionRunService.cs

```diff
@@ -169,7 +169,7 @@ private IQueryable<MissionRun> GetMissionRunsWithSubModels()
                 .ThenInclude(robot => robot.Model)
                 .Include(missionRun => missionRun.Tasks)
                 .ThenInclude(task => task.Inspections)
-                .Where((m) => accessibleInstallationCodes.Result.Contains(m.Area.Installation.InstallationCode)); ;
+                .Where((m) => m.Area == null || accessibleInstallationCodes.Result.Contains(m.Area.Installation.InstallationCode.ToUpper())); ;
```

Comment:
Will KAA users get all missions from JSV without an area?

Reply:
If a missionRun does not have an area it also does not have an installation. We can enforce the use of Area in missionRun maybe but I think this will require some refactoring for the localisation code